ClientSide(FrontEnd)/clay_/src/components/signup.css

```diff
@@ -0,0 +1,85 @@
+
+.starting{
```

Comment:
use proper naming convention as className

Reply:
i changed naming convention in current branch